android-core/src/main/java/com/mparticle/MParticle.java

```diff
@@ -639,7 +639,7 @@ public void enableLocationTracking(@NonNull String provider, long minTime, long
                     //noinspection MissingPermission
                     locationManager.requestLocationUpdates(provider, minTime, minDistance, mLocationListener);
                     locationTrackingEnabled = true;
-                }catch (SecurityException se) {
+                } catch (SecurityException se) {
```

Comment:
this should be ignored 

`} catch (SecurityException ignored) { }`

Reply:
👍